default.css

```diff
@@ -131,22 +419,187 @@ table.table thead .sorting_desc_disabled {
   *cursor: hand;
 }
 
-table.table thead .sorting { background: url('datatables/images/sort_both.png') no-repeat center right; }
-table.table thead .sorting_asc { background: url('datatables/images/sort_asc.png') no-repeat center right; }
-table.table thead .sorting_desc { background: url('datatables/images/sort_desc.png') no-repeat center right; }
+table.table thead .sorting {
+  background: url("datatables/images/sort_both.png") no-repeat center right;
+}
+table.table thead .sorting_asc {
+  background: url("datatables/images/sort_asc.png") no-repeat center right;
+}
+table.table thead .sorting_desc {
+  background: url("datatables/images/sort_desc.png") no-repeat center right;
+}
 
-table.table thead .sorting_asc_disabled { background: url('datatables/images/sort_asc_disabled.png') no-repeat center right; }
-table.table thead .sorting_desc_disabled { background: url('datatables/images/sort_desc_disabled.png') no-repeat center right; }
+table.table thead .sorting_asc_disabled {
+  background: url("datatables/images/sort_asc_disabled.png") no-repeat center
+    right;
+}
+table.table thead .sorting_desc_disabled {
+  background: url("datatables/images/sort_desc_disabled.png") no-repeat center
+    right;
+}
 
 /* This class is applied to the DataTables search box by our own code once it renders */
 input.search {
-  margin-left: 5px;
-  width: 150px;
+  width: 200px;
   display: inline;
+  height: 36px;
 }
 
 /* Bootstrap overrides *******************************************************/
 .tooltip-inner {
   /* let text wrap in tooltips */
   white-space: pre-wrap;
 }
+
+.thead-sticky {
+  position: sticky;
+  top: 0;
+  z-index: 2;
+  background: #ffffff;
+  margin-bottom: 10px;
+}
+
+.active-line {
+  background-color: #f4f6ff;
+  color: #000000;
+}
+
+a:hover {
+  background-color: #fafafa;
+}
+
+.table-container {
+  overflow-x: auto;
+  overflow-y: auto;
+  width: 100%;
+  max-height: 100vh;
+  overflow: auto;
+  border-top: 1px solid rgba(0, 0, 0, 0.12);
+}
+
+.table-with-filters {
+  display: flex;
+  flex-direction: column;
+  padding-top: 64px;
+}
+
+.table {
+  width: 100% !important;

```

Comment:
I think it's duplicate. There is already some styles for [table](https://github.com/doitintl/gcpinstances.info/pull/38/files#diff-bec084bdca5e9e44aaac6baa3a0dfb80524284f7e187b7b3f4d7fd667879724eR394) here

Reply:
@hemlata-pathak fixed